docs/main/docs/components/become-an-executor.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 2
+---
+
+# How to Become an Executor
+
+This documentation covers how to become an Executor on the t3rn network. It assumes that you have accounts with the correct funds for the desired blockchains, and that you have Substrate installed, which is needed for generating keys with subkey; we will share the resources to complete these steps.
+
+## Step 1 - Clone repo
+
+Clone the [t3rn repo](https://github.com/t3rn/t3rn/tree/development/client/packages/executor): `git clone git@github.com:t3rn/t3rn.git`.
+
+<p align="center">
+    <img height="150" src="/img/clone-executor-repo.png?raw=true"/>
+</p>
+
+## Step 2 - Install dependencies
+
+`pnpm install` in the above directory (t3rn/client/packages/executor/).
+
+<p align="center">
+    <img height="150" src="/img/install-executor-dep.png?raw=true"/>
+</p>
+
+## Step 3 - Configure your Executor
+
+Configure your Executor by editing `.envrc-example` and changing the values to your liking, renaming it to `.envrc` and using with [direnv](https://github.com/direnv/direnv#how-it-works).
+
+<p align="center">
+    <img height="150" src="/img/envrc.png?raw=true"/>
+</p>
+
+## Step 4 - Run Executor
+
+If you’ve configured your `.envrc` file, then simply run `pnpm start`.
+
+<p align="center">
+    <img height="150" src="/img/run-executor.png?raw=true"/>
+</p>
+
+You can also run the command below and add your keys as in the example, if you don’t want to store them unencrypted on the server
```

Comment:
I think `LOG_PRETTY=true CIRCUIT_WS_ENDPOINT=wss:/rpc.t0rn.io ETHEREUM_PRIVATE_KEY=0x1234 CIRCUIT_SIGNER_KEY=0x1234 RELAYCHAIN_SIGNER_KEY=0x1234 pnpm start` should be default way to run for people because not everybody wants to install/use direnv.

Reply:
I've changed the order on them so the suggested default comes first